### PR TITLE
updated sublime-text-dev (3099)

### PR DESCRIPTION
**Pull request by ravecode**

### CODE REVIEW

File: Casks/sublime-text-dev.rb [hunk 1]
```diff
@@ -1,11 +1,12 @@
+
```
Comment (vitorgalvao) [inline Casks/sublime-text-dev.rb]: This empty line must go.

File: Casks/sublime-text-dev.rb [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 cask 'sublime-text-dev' do
-  version '3095'
-  sha256 'f03b7af33cc20dbff863b35016f77803239168e2bd3691b756a3e4729cb5e5c2'
+  version '3099'
+  sha256 '779c6a7601fed19867ed285b390fb960fe64473f4112bc2822b479310f7f6665'
 
-  # rackcdn.com is the official download host per the vendor homepage
-  url "http://c758482.r82.cf2.rackcdn.com/Sublime%20Text%20Build%20#{version}.dmg"
+  # download.sublimetext.com is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/sublime-text-dev.rb]: This comment must go, it’s only needed when hosts differ.

**Comment by ravecode** [comment]
Issues have been fixed.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by ravecode** [comment]
Commits have been squashed.

**Pull request merged**